factory/network/networkComponents.go

```diff
@@ -136,6 +143,11 @@ func (ncf *networkComponentsFactory) Create() (*networkComponents, error) {
 		return nil, fmt.Errorf("%w for the full archive network holder", err)
 	}
 
+	lightClientNetworkComp, err := ncf.createLightClientNetworkHolder(peersRatingHandler)
```

Comment:
don't forget to call Close on this new messenger instance, similar with L379-L383

also, it should be added on CheckSubcomponents, part of `networkComponentsHandler.go`, similar with L92-L94

also we'll need a getter for this messenger, as it is on L221-L230, but this can be done in further PRs where we'll actually send messages on this network

Reply:
added.